### DISCUSSION TITLE
Scattering length data?

### DISCUSSION THREAD

**Discussion by XiaoCasd** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#161] [Ideas]
This may sound futuristic, but I am wondering if it is possible to implement a scattering length function, such as 
`scattering_length(atom1, atom2, atom1_internal_state, atom2_internal_state, magnetic_field)`

This functionality will be handy for ultracold atomic experiments. Because all the Feshbach resonance/scattering length data are scattered, one needs to search the literature intensively to find the right number. It will be great if one function can handle it. Would love to contribute to this functionality if people believe this is a good feature.

**Comment by nikolasibalic** [comment]
HI @XiaoC-671 , sorry for delay in response.

It sounds like a great idea. :star_struck:  I haven't been calculating scattering lenghts myself before, so physics wise I cannot help with some insight (would have to start from scratch myself). But in principle it should be possible. And we certanly also add some tables compiled from literature.

Since it is two-atom functionality, it can go under new file like `calculations_atom_scattering.py`.

Let me know how you wish to proceed and how I can help!
I am excited about this! 
Nikola
